src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,40 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* Globalnet can now be specifically disabled for a cluster on join via the `--globalnet` flag.
```

Comment:
By default globalnet is disabled and if required, it can be enabled using the --globalnet flag.

Reply:
Hmm, no she means something different, if you enable globalnet for all, then joining a cluster it can be manually disabled. But anyway, that's a workaround for an specific use case, that may be we should not publicize much?

I would remove the line altogether.

Reply:
yeah it's more or less an isolated use case - I agree we can omit it.

Reply:
Omitted.